doc/reduce-traffic.md

```diff
@@ -26,7 +26,7 @@ calculating the target.
 
 ## 2. Disable "listening" (`-listen=0`)
 
-Disabling listening will result in fewer nodes connected (remember the maximum of 8
+Disabling listening will result in fewer nodes connected (remember the maximum of 10
```

Comment:
Line `6` also has the outdated number.

Reply:
Yup, might want to change the number of peers (and the composition: 8 full outgoing connections and two blocks-only).

Reply:
> Yup, might want to change the number of outgoing peers there (and mention the composition: 8 full outgoing connections and two blocks-only).

This would probably mean that the max connection now is 127, since we have 10 outgoing and 117 incoming?

Reply:
No, DEFAULT_MAX_PEER_CONNECTIONS is still 125